### PR TITLE
Clean up and update to pymongo 4 "utils" scripts

### PR DESCRIPTION
**Pull request by ppigazzini**

### CODE REVIEW

**Comment by tomtor** [comment]
@ppigazzini  `index_pending.py` could be removed, `create_indexes.py` does all that's needed?

**Comment by ppigazzini** [comment]
Yes, it works fine in the server setup script
```bash
# add mongodb indexes
sudo -i -u ${user_name} << 'EOF'
${VENV}/bin/python3 ${HOME}/fishtest/server/utils/create_indexes.py actions flag_cache pgns runs users
EOF
```

File: server/utils/upgrade.py [hunk 1]
```diff
@@ -82,6 +82,89 @@ def show(p):
 
 worker_dict = {}
 
+<<<<<<< HEAD
+=======
```
Comment (dav1312) [inline server/utils/upgrade.py:86]: Merge conflict?

File: server/utils/upgrade.py [hunk 2]
```diff
@@ -82,6 +82,89 @@ def show(p):
...
+
+        # Two other things that changed
+        if "ARCH" not in worker_info:
+            if "ARCH" in task:
+                worker_info["ARCH"] = task["ARCH"]
+                del task["ARCH"]
+            if "nps" in task:
+                worker_info["nps"] = task["nps"]
+                del task["nps"]
+>>>>>>> 1f00e05... Clean up utils scripts and update to pymongo 4.x
```
Comment (dav1312) [inline server/utils/upgrade.py:167]: Same?

**Pull request closed without merging**